third_party/silabs/silabs_board.gni

```diff
@@ -97,8 +97,20 @@ if (silabs_board == "BRD4304A") {
   silabs_family = "efr32mg12"
   silabs_mcu = "EFR32MG12P433F1024GM68"
 } else if (silabs_board == "BRD4325B") {
-  silabs_family = "SiWx917"
```

Comment:
I would leave it as is. It's a regression. Who cares if it has dual flash or single flash, it's still a 917. so the family should stay the same. It's supposed to be generic and englobe all of the potential variant